### ISSUE TITLE
Radiobutton not announced on iOS

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#42794]
On Android, Radio and RadioListTile widgets are given a semantic hint of "radio button". However, this doesn't seem to happen on iOS. As it is, iOS users have no way to know the elements are radio buttons (and not checkboxes).

repro case:

```dart
import 'package:mobile.flutter.google_material.components/google_material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'radio issue repro',
      theme: GoogleMaterial.theme(),
      home: Scaffold(
        body: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Radio(),
            RadioListTile(
              title: Text('item 1'),
            ),
            RadioListTile(
              title: Text('item 2'),
            ),
          ],
        ),
      ),
    );
  }
}
```

on Android, the first RadioListTile is announced as "checked, item 1, radio button, disabled"
on iOS, that same RadioListTile is announced as "selected, item 1, dim"

**Comment by mehmetf** [comment]
Filed in duplicate.
